exporter/trace/cloudtrace.go

```diff
@@ -81,6 +84,14 @@ func WithProjectID(projectID string) func(o *options) {
 	}
 }
 
+// WithDestinationProjectQuota enables per-request usage of the destination
+// project's quota. For example, when setting gcp.project.id on a span.
```

Comment:
gcp.project.id is a resource attribute, rather than a span attribute, right?

Reply:
Yeah, updated